sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.kt

```diff
@@ -59,11 +60,17 @@ class AuthorizingPaymentActivity : AppCompatActivity() {
         supportActionBar?.title = uiCustomization.uiCustomization.toolbarCustomization?.headerText
             ?: getString(R.string.title_authorizing_payment)
 
-        if (verifier.verifyExternalURL(verifier.authorizedURL)) {
-            openDeepLink(verifier.authorizedURL)
+        val authUrl = verifier.authorizedURLString
+        // check for legacy payments that require web view
+        if (authUrl.endsWith("/pay")) {
```

Comment:
Is this the only one possibility to verify legacy url?

Reply:
Yes from what @nuxzero described in the ticket.

Reply:
I saw 2 ways from the ticket, which of the following options is potentially more precise?
- The string end with `/authorize` for payment v2
- The string end with `/pay` for legacy payments

Reply:
@ajzkk  Some legacy payments support payment v2. Payment v2 accepts receiving an authorization request and will not reply with `404`. V2 end with `/authorize` which we do not want to change. On the other hand some other legacy payments do not support payment v2. These urls end with `/pay` and these are the urls that we want to specifically handle